cmd/podman/containers/create.go

```diff
@@ -286,6 +286,12 @@ func CreateInit(c *cobra.Command, vals entities.ContainerCreateOptions, isInfra
 			}
 			vals.Env = env
 		}
+
+		if tFlag, err := c.Flags().GetBool("tty"); err != nil {
+			return vals, fmt.Errorf("Failed to get tty flag: %w", err)
+		} else if !tFlag {
```

Comment:
No need for else here. Would it be better to just remove the automatic TERM setting rather then doing this.